examples/best_of_n.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<p>Maybe add a few sentences to explain the motivation of the notebook and what does it showcases</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lvwerra/trl/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>